Gemfile

```diff
@@ -22,6 +22,6 @@ group :test do
   gem 'rubocop-performance', require: false
 
   platform :mri, :truffleruby do
-    gem 'liquid-c', github: 'Shopify/liquid-c', ref: 'master'
+    gem 'liquid-c', github: 'Shopify/liquid-c', ref: 'inline-comment'
```

Comment:
What's is the plan here? Will you merge it remove current change so it point to master?

Reply:
The liquid-c change can be safely merged first, then I'll update this to point back to master before merging.